src/ChatView.module.css

```diff
@@ -59,6 +59,7 @@
 }
 
 .right {
+  text-align: right;
```

Comment:
Current alignment for long messages is to the left

![Screenshot 2025-01-16 at 10 59 54 AM](https://github.com/user-attachments/assets/854cec04-594b-4bac-9319-ecfb202afab3)